.travis.yml

```diff
@@ -10,6 +10,4 @@ install:
   - "pip install --upgrade setuptools pip"
   - "pip install 'pytest==3.2.5' mock"
   - "pip install -e .[openstack,ec2,daemon]"
-script:
-  - export BOTO_CONFIG=/dev/null
```

Comment:
This should not be removed - fixes issues with testing on Travis CI shipping an invalid Boto config.